Jenkinsfile

```diff
@@ -55,4 +55,10 @@ node("primary") {
       }
     }
   }
+
+  stage("Publish") {
+    if (isReleaseBranch()) {
```

Comment:
You should probably have the `stage` inside this, otherwise branch builds will be very confusing as it'll look like they've been published. eg:

```
  if (isReleaseBranch()) {
    stage("Publish") {
      sh 'yarn publish'
    }
  }
```